mlir/Makefile

```diff
@@ -75,6 +76,10 @@ dialects:
 		-DCMAKE_CXX_COMPILER_LAUNCHER=ccache
 
 	cmake --build $(DIALECTS_BUILD_DIR) --target check-dialects quantum-lsp-server
+	find $(DIALECTS_BUILD_DIR)/python_packages/ -type l -exec sed --in-place 's/from \._ods_common /from jaxlib.mlir.dialects._ods_common /g' {} \;
```

Comment:
Can we add a patch to fix `_ods_common` rather than replace all occurrences of it?

```
--- _ods_common.py
+++ _ods_common.py
@@ -4,7 +4,7 @@
 
 # Provide a convenient name for sub-packages to resolve the main C-extension
 # with a relative import.
-from .._mlir_libs import _mlir as _cext
+from jaxlib.mlir._mlir_libs import _mlir as _cext
 from typing import Sequence as _Sequence, Union as _Union
```

Reply:
Speaking of patches, do we know how brittle these are? (vs a string search for that particular line to replace)
Say if the relevant line moved down by one line, would the patch break? Or would the line itself need modification before the patch breaks?

Reply:
In my understanding they follow the same strategy that git does.

Reply:
From the manual:

```
       With context diffs, and to a lesser extent with normal diffs, patch can detect when the line numbers mentioned in the patch are incorrect, and attempts to find the  correct  place  to  apply
       each  hunk  of  the patch.  As a first guess, it takes the line number mentioned for the hunk, plus or minus any offset used in applying the previous hunk.  If that is not the correct place,
       patch scans both forwards and backwards for a set of lines matching the context given in the hunk.  First patch looks for a place where all lines of the context match.  If no such  place  is
       found,  and  it's  a context diff, and the maximum fuzz factor is set to 1 or more, then another scan takes place ignoring the first and last line of context.  If that fails, and the maximum
       fuzz factor is set to 2 or more, the first two and last two lines of context are ignored, and another scan is made.  (The default maximum fuzz factor is 2.)

```

Reply:
Nice thanks!